source/contributing/index.md

```diff
@@ -0,0 +1,42 @@
+---
```

Comment:
If this _contributing guide_ applies only to this repo, I would suggest to use the existing convention of having a `CONTRIBUTING.md` file in the repo's root, as this is _recognised_ by GitHub (as well as other collaboration tools).

Reply:
👍